### ISSUE TITLE
ac-options function with params

### ISSUE THREAD

**Issue by shotap** [john-oc/acute-select#41]
Hi,
I'm trying to do something like this:
```javascript
ac-options="l as l in loadData(1)"
```
When I'm adding a param to the function the directive isn't working, there is an option to do it?

**Comment by john-oc** [comment]
No, sorry. The directive passes a set list of parameters (callback function, search text and item offset). You could set a value on the scope, though, and pick that up in the loadData function.

**Comment by shotap** [comment]
OK. i will create a PR soon...
